app/AcquireIntersonArrayBMode.cxx

```diff
@@ -147,13 +147,18 @@ int main( int argc, char * argv[] )
     std::cerr << "Invalid requested depth for probe." << std::endl;
     }
 
+  const double ns = ContainerType::MAX_SAMPLES; // number of samples
+  const double fs = frequencies[ frequencyIndex ] / 1000; // [kHz]=[samples/ms] - frequency per sample
```

Comment:
`frequencies` is the excitation center frequency, which is different from the sampling frequency. The sampling frequency is a fixed value for this probe.